wordpress-mobile/WordPress-Editor-iOS
# The title field no longer adds spaces when switching edit modes.

Fixes #766 .

Also makes the title correction settings follow the default by the user.

**How to test:**
1. Run the editor.
2. Switch to HTML mode.  Notice the title field has no spaces at the end of it.
3. Switch to visual mode.  Same verification.
4. Repeat at least 2 more times from step 2 (since the first time it was working fine before too).

/cc @bummytime for review.
